worlds/hk/docs/setup_en.md

```diff
@@ -44,9 +46,8 @@ website to generate a YAML using a graphical interface.
    * If you are waiting for a countdown then wait for it to lapse before hitting Start.
    * Or hit Start then pause the game once you're in it.
    
-## Commands
+## Hints and other commands
 While playing the multiworld you can interact with the server using various commands listed in the 
 [commands guide](/tutorial/Archipelago/commands/en). As this game does not have an in-game text client at the moment,
 You can optionally connect to the multiworld using the text client, which can be found in the 
-[main Archipelago installation](https://github.com/ArchipelagoMW/Archipelago/releases) as Archipelago Text Client to
-enter these commands.
+[main Archipelago installation](https://github.com/ArchipelagoMW/Archipelago/releases) as Archipelago Text Client to enter these commands.
```

Comment:
This sentence as a whole is a bit awkward with "to enter these commands" coming after two clauses, but it's fine
```suggestion
[main Archipelago installation](https://github.com/ArchipelagoMW/Archipelago/releases) listed as "Archipelago Text Client", to enter these commands.
```